subprojects/docs/docs.gradle

```diff
@@ -158,6 +158,11 @@ String snippetId(File snippetLocation) {
 // TODO (donat) verify the content of the samples index
 
 List ignoredSnippets = [
+    snippetId(file('src/snippets/compositeBuilds/basic')), // promoted sample
+    snippetId(file('src/snippets/compositeBuilds/declared-substitution')), // promoted sample
+    snippetId(file('src/snippets/compositeBuilds/hierarchical-multirepo')), // promoted sample
+    snippetId(file('src/snippets/compositeBuilds/plugin-dev')), // promoted sample
```

Comment:
I think we can probably delete SamplesCompositeBuildIntegrationTest.groovy that needs these snippets, since we have other integration tests that cover these areas.

Reply:
I've converted all but two test cases into exemplar tests, then deleted the `SamplesCompositeBuildIntegrationTest`